### ISSUE TITLE
Validation step metrics

### ISSUE THREAD

**Issue by jordancaraballo** [nasa-nccs-hpda/above-shrubs#6]
We need to add more metrics for the validation of the test dataset. RMSE, R2, and MAE are in the works. RMSE needs to be in-house developed.

**Comment by mmacander** [comment]
r2_score is a good regression metric and different than the standard correlation coefficient from a linear regression version 
[scikit](https://scikit-learn.org/stable/modules/generated/sklearn.metrics.r2_score.html).
